com.unity.render-pipelines.high-definition/Editor/AssetProcessors/ModelPostProcessor.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using UnityEngine;
+using UnityEngine.Rendering.HighDefinition;
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    class ModelPostprocessor : AssetPostprocessor
+    {
+        static void AddAdditionalData<T, AdditionalT>(GameObject go, Action<AdditionalT> initDefault = null)
+            where T : Component
+            where AdditionalT : Component
+        {
+            var components = go.GetComponentsInChildren(typeof(T), true);
+            foreach (var c in components)
+            {
+                if (!c.TryGetComponent<AdditionalT>(out _))
+                {
+                    var hd = c.gameObject.AddComponent<AdditionalT>();
+                    if (initDefault != null)
+                        initDefault(hd);
+                }
+            }
+        }
+
+        void OnPostprocessModel(GameObject go)
+        {
+            AddAdditionalData<Camera, HDAdditionalCameraData>(go, HDAdditionalCameraData.InitDefaultHDAdditionalCameraData);
```

Comment:
So if I read this correctly, as long as we have the HDRP package present then importing a model will add HD additional data components. 
If we remove the package then what happens, is it okay to leave them?

Reply:
Yes. Did you test with HDRP package installed but the RP in use is Universal? Could it work in this case?

Reply:
Also, Universal have the same additional component for their Camera. What happens if both Universal and HD is installed and the prefab as a Camera, a UniversalAdditionalCameraData and a HDAdditionalCameraData?

Reply:
You are right, I didn't think about it
Also since the bug also happens on URP, i need to fix it for both

Reply:
So now if a pipeline is present in the project it will always add the additional data, otherwise if you switch active pipeline after model import, the object is not reimported and you still have the errors